lukasloetkolben/openpilot
# added usa 2018 model 3 vehicle CAN (engine) to fingerprints.py [model3_stock_acc]

**_Same as #3 for model3_stock_acc branch_**

**Car**
USA 2018 Tesla Model 3 LR (AP2.5 Computer)

**Route**
b0af2fff166d80fe/00000014--34345db9f1/0 [done on model3 branch]

**Test Plan**

1.   Observe unsupported car message before this change
2.   Add new fingerprint to engine ECU ID based on first segment route's carFw
3.   Reboot Comma 3, Observe working OP

**CarFw:**

```
 'carFw': [{'address': 1840,
                          'brand': 'tesla',
                          'bus': 0,
                          'ecu': 'eps',
                          'fwVersion': b'TeM3_E014p10_0.0.0 (16),E014.17.00',
                          'logging': False,
                          'obdMultiplexing': True,
                          'request': [b'>\x00', b'"\xf1\x95'],
                          'responseAddress': 1848,
                          'subAddress': 0},
                         {'address': 1542,
                          'brand': 'tesla',
                          'bus': 1,
                          'ecu': 'engine',
                          'fwVersion': b'\x01\x00\x05\x18A\x00\x02\x00\x00\x00\x00\x00\x00\x00\x00\x00\x000\x91',
                          'logging': False,
                          'obdMultiplexing': False,
                          'request': [b'>\x00', b'"\xf1\x81'],
                          'responseAddress': 1558,
                          'subAddress': 0}],
```
